metadata-node-red-app

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "node-red-app",
+    "crn:v1:staging:public:resource-catalog::a/bef71ea83f3d0c18eb701256cf445eb3::template:node-red-app",
+    "url": "https://globalcatalog.test.cloud.ibm.com/api/v1/node-red-app?complete=true",
+    "name": "Node-Red-Starter",
+    "kind": "template",
+    "overview_ui": {
+        "en": {
+            "display_name": "Node-Red-Starter",
+            "long_description": "This starter kit comes pre-configured to create your own Node-RED based application that can be quickly deployed to IBM Cloud. Add services, generate and download the code, use the IBM Cloud Developer Tools CLI to run and debug locally, then deploy to Kubernetes, Cloud Foundry, or a DevOps Pipeline.",
```

Comment:
this description doesnt sound accurate or interesting, you should spend a little time reworking it. for example it doesnt mention that you do development in the app at all.

Reply:
Since the description was taken from the existing NodeRED boilerplate, we can keep it for now and update it later. We'll need more info from Nick O'Leary.